javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/testingclasses/TargetType.java

```diff
@@ -0,0 +1,25 @@
+package com.github.javaparser.symbolsolver.testingclasses;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import java.lang.invoke.LambdaMetafactory;
+import java.lang.invoke.MethodHandle;
+import java.util.concurrent.TimeUnit;
+
+@interface NestedAnnotation {
```

Comment:
Can you reuse the Annotations.java.txt test resource to declare this annotation?